src/filters/index.js

```diff
@@ -18,7 +18,7 @@ export const formatBech32 = (address, longForm = false, length = 4) => {
 
 export const resolveValidatorName = (address, validators) => {
   if (validators[address]) {
-    return validators[address].moniker
+    return validators[address].description.moniker
```

Comment:
The validator object is a flat structure from GraphQL now.

You can see the TransactionItems won't fill out properly.